app/routes/login.js

```diff
@@ -3,26 +3,25 @@ import { inject as service } from '@ember/service';
 import { isPresent } from '@ember/utils';
 import { Promise } from 'rsvp';
 import EmberConfig from 'ilios/config/environment';
-import UnauthenticatedRouteMixin from 'ember-simple-auth/mixins/unauthenticated-route-mixin';
 
-export default Route.extend(UnauthenticatedRouteMixin, {
-  currentUser: service(),
-  fetch: service(),
-  iliosConfig: service(),
-  session: service(),
+export default class LoginRoute extends Route {
+  @service currentUser;
+  @service fetch;
+  @service iliosConfig;
+  @service session;
 
-  noAccountExistsAccount: null,
-  noAccountExistsError: false,
+  noAccountExistsAccount = null;
+  noAccountExistsError = false;
 
-  beforeModel(transition){
-    this._super(transition);
```

Comment:
I'm not sure what this did, but wanted to make sure you dropped it on purpose.

Reply:
hmm, according to the docs, it was necessary to invoke the parent method in order to prevent infinite redirects.

```
Checks whether the session is authenticated and if it is aborts the current
    transition and instead transitions to the
    {{#crossLink "UnauthenticatedRouteMixin/routeIfAlreadyAuthenticated:property"}}{{/crossLink}}.

   __If `beforeModel` is overridden in a route that uses this mixin, the route's
   implementation must call `this._super(...arguments)`__ so that the mixin's
   `beforeModel` method is actually executed.
```

supposedly, this can be ditched now without replacement. in click-testing, i was unable to trigger an infinite redirect.